### PR TITLE
Enhance install.sh to help with k3s upgrade and uninstall scenarios

### PR DESCRIPTION
**Pull request by Oats87**
Enhance install.sh to help with k3s upgrade and uninstall scenarios for k3s-selinux

https://github.com/rancher/k3s/issues/2517
https://github.com/rancher/k3s/issues/2518

Signed-off-by: Chris Kim <oats87g@gmail.com>

### CODE REVIEW

File: install.sh [hunk 1]
```diff
@@ -661,6 +673,11 @@ rm -rf /var/lib/rancher/k3s
 rm -rf /var/lib/kubelet
 rm -f ${BIN_DIR}/k3s
 rm -f ${KILLALL_K3S_SH}
+
+if [ -r /etc/redhat-release ] || [ -r /etc/centos-release ] || [ -r /etc/oracle-release ]; then
```
Comment (dweomer) [inline install.sh:677]:
```suggestion
if type yum >/dev/null 2>&1; then
```

Comment (Oats87) [reply install.sh:677]: I was trying to be a little more precise with this, I think there are other distros that may have `yum` installed. I guess just  this command is pretty blind and doesn't matter if it fails

File: install.sh [hunk 2]
```diff
@@ -661,6 +673,11 @@ rm -rf /var/lib/rancher/k3s
 rm -rf /var/lib/kubelet
 rm -f ${BIN_DIR}/k3s
 rm -f ${KILLALL_K3S_SH}
+
+if type yum >/dev/null 2>&1; then
+    yum remove -y k3s-selinux
+    rm -f /etc/yum.repos.d/rancher-k3s-common*.repo
```
Comment (brandond) [inline install.sh:679]: Does this throw an error if there isn't a repo file?

**Review by brandond — APPROVED** [review]

File: install.sh (hunk 2 above)
Comment (Oats87) [reply install.sh:679]: Neither the `yum remove -y k3s-selinux` or the `rm -f` return errors even when there are no files/RPMs that match for removal, i.e.
```
[root@ck-centos7-0 ~]# ls
anaconda-ks.cfg
[root@ck-centos7-0 ~]# echo $?
0
[root@ck-centos7-0 ~]# rm -f lol
[root@ck-centos7-0 ~]# echo $?
0
[root@ck-centos7-0 ~]# yum remove -y k3s-selinux
Loaded plugins: fastestmirror
No Match for argument: k3s-selinux
No Packages marked for removal
[root@ck-centos7-0 ~]# echo $?
0
[root@ck-centos7-0 ~]#
```

**Review by briandowns — APPROVED** [review]
LGTM

**Pull request merged**